File: LogicaDeProgramacaoEclipse(Java)/LogicaProgramacaoJava(Eclipse)/exerciciosJava/src/exerciciosFaccat/Exercicio25.java
package exerciciosFaccat;

import java.util.Scanner;

/*25) Faça um algoritmo para ler: número da conta do cliente, saldo, débito e crédito. Após, calcular e
escrever o saldo atual (saldo atual = saldo - débito + crédito). Também testar se saldo atual for maior
ou igual a zero escrever a mensagem 'Saldo Positivo', senão escrever a mensagem 'Saldo Negativo'. */

public class Exercicio25 {
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);

		float numeroConta;

		float saldo, debito, credito, saldoAtual;

		System.out.println("Digite o numero de Conta do cliente:");
		numeroConta = sc.nextFloat();
		System.out.println("Digite o saldo do cliente:");
		saldo = sc.nextFloat();
		System.out.println("Digite o debito do cliente:");
		debito = sc.nextFloat();
		System.out.println("Digite o credito do cliente:");
		credito = sc.nextFloat();
		
		saldoAtual = saldo - debito + credito;
		System.out.println("O saldo atual é: " + saldoAtual);

		if (saldoAtual >= 0) {
			System.out.println("Saldo Atual Positivo");
		} else {
			System.out.println("Saldo Atual Negativo");
		}
		
		sc.close();
		
	}
}

File: LogicaDeProgramacao(JavaScript)/Exercicios 5 até o 39 Apostila Faccat/js/exercicio25.js
/*25) Faça um algoritmo para ler: número da conta do cliente, saldo, débito e crédito. Após, calcular e
// escrever o saldo atual (saldo atual = saldo - débito + crédito). Também testar se saldo atual for maior
// ou igual a zero escrever a mensagem 'Saldo Positivo', senão escrever a mensagem 'Saldo Negativo'. */


var saldoAtual = 0

   var numeroConta = parseInt(prompt("Digite o numero de Conta do cliente:"))
   var saldo = parseFloat(prompt("Digite o saldo do cliente:"))
   var debito = parseFloat(prompt("Digite o debito do cliente:"))
   var credito = parseFloat(prompt("Digite o credito do cliente:"))

   saldoAtual = saldo - debito + credito
   alert("O saldo atual é: "+ saldoAtual)

   if (saldoAtual >= 0) {
      alert("Saldo Atual Positivo")
   } else {
      alert("Saldo Atual Negativo")
   }